.github/workflows/ci.yml

```diff
@@ -588,6 +588,16 @@ jobs:
           echo "Tests: $TESTS"
           ./mvnw test ${{ env.SUREFIRE_RETRY }} -Pauth-server-quarkus -Pdb-${{ matrix.db }} "-Dwebdriver.chrome.driver=$CHROMEWEBDRIVER/chromedriver" -Dtest=$TESTS -pl testsuite/integration-arquillian/tests/base 2>&1 | misc/log/trimmer.sh
 
+      - name: Run cluster JDBC_PING2 smoke test
```

Comment:
I have added this to the existing `store-integration-tests` job for simplicity, but maybe it's better to create a dedicated job as this is not really related to store functionality?